ros_ign_interfaces/msg/Color.msg

```diff
@@ -0,0 +1,6 @@
+std_msgs/Header header
+
+float32 r       # Red component
+float32 g       # Green component
+float32 b       # Blue component
+float32 a       # Alpha component
```

Comment:
I think we should use ROS's `ColorRGBA` message instead of creating a new one:

https://github.com/ros2/common_interfaces/blob/galactic/std_msgs/msg/ColorRGBA.msg